rest-api/dao/participant_summary_dao.py

```diff
@@ -381,30 +400,32 @@ def update_enrollment_status(self, summary):
       consent, summary)
     summary.enrollmentStatus = enrollment_status
 
-  def calculate_enrollment_status(self, consent_for_study_enrollment_and_ehr,
+  def calculate_enrollment_status(self, consent,
                                   num_completed_baseline_ppi_modules,
                                   physical_measurements_status,
                                   samples_to_isolate_dna):
-    if consent_for_study_enrollment_and_ehr:
+    if consent:
       if (num_completed_baseline_ppi_modules == self._get_num_baseline_ppi_modules() and
           physical_measurements_status == PhysicalMeasurementsStatus.COMPLETED and
           samples_to_isolate_dna == SampleStatus.RECEIVED):
         return EnrollmentStatus.FULL_PARTICIPANT
       return EnrollmentStatus.MEMBER
     return EnrollmentStatus.INTERESTED
 
-  def calculate_member_time(self, consent_for_study_enrollment_and_ehr, participant_summary):
-    if consent_for_study_enrollment_and_ehr and \
-      participant_summary.enrollmentStatusMemberTime is not None:
+  def calculate_member_time(self, consent, participant_summary):
+    if consent and participant_summary.enrollmentStatusMemberTime is not None:
       return participant_summary.enrollmentStatusMemberTime
-    elif consent_for_study_enrollment_and_ehr:
+    elif consent:
+      if participant_summary.consentForElectronicHealthRecords is None and \
```

Comment:
Are these conditions guaranteed to be mutually exclusive? Can they not consent to both? If not, does this PR intentionally prefer non-DV EHR?

Reply:
as stated above, consent to both is a very real possibility and either is acceptable.